pycon/settings/base.py

```diff
@@ -359,3 +361,7 @@ def env_or_default(NAME, default):
 # This can be removed in 1.8, the warning has been
 # removed.
 TEST_RUNNER = 'django.test.runner.DiscoverRunner'
+
+# Need to switch from the now-default serializer or OAuth2 breaks trying
+# to serialize a datetime
+SESSION_SERIALIZER='django.contrib.sessions.serializers.PickleSerializer'
```

Comment:
I haven't noticed any issues running locally. And the SESSION_SERIALIZER docs don't appear to have any warnings about changing it.